cms/djangoapps/contentstore/tests/test_transcripts_utils.py

```diff
@@ -184,6 +186,17 @@ def clear_subs_content(self, youtube_subs):
         for subs_id in youtube_subs.values():
             self.clear_sub_content(subs_id)
 
+    def generate_random_string(self, string_length=100):
```

Comment:
Just a choice, no reason to have it 100. If you find any other appropriate value, please guide.